test/CMakeLists.txt

```diff
@@ -25,7 +25,7 @@ file(COPY
 # Invoked as add_coverage_test("SomeName" sources...)
 function(add_coverage_test Name)
     add_executable(${Name} ${ARGN})
-    target_compile_options(${Name} PRIVATE -g -Og)
+    target_compile_options(${Name} PRIVATE -g -Og -Wno-deprecated-declarations)
```

Comment:
This _could_ be `-Wno-error=deprecated-declarations` instead to preserve the warnings when the test code is built.